go/oasis-node/cmd/debug/txsource/workload/runtime.go

```diff
@@ -112,15 +113,22 @@ func (r *runtime) submitRuntimeRquest(ctx context.Context, rtc runtimeClient.Run
 		"request", req,
 	)
 
-	// Wait for a maximum of 30 seconds as invalid submissions may block
-	// forever.
-	submitCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
-	out, err := rtc.SubmitTx(submitCtx, rtx)
+	// Retry in case a runtime node is just getting restarted.
```

Comment:
This is interesting, why does the workload need to handle retries? Shouldn't the client node handle this correctly as client nodes are not restarted? What exactly happens, does it just time out? If so, we may want to just increase the timeout value (to 120s?) instead of doing retries here. This is because retrying here could mask bugs in the client node not handling errors/retries correctly.

Reply:
hm, pretty sure it failed on the first compute node restart when i tried it the first time. iirc it timed out - didn't look to much into it - but i will re-check.

Reply:
Maybe try with an increased timeout (e.g., 120s).

Reply:
yeah seems to work. will revert the retries.